cmd/koord-descheduler/app/options/scheme.go

```diff
@@ -19,6 +19,7 @@ package options
 import (
 	appsv1alpha1 "github.com/openkruise/kruise-api/apps/v1alpha1"
 	appsv1beta1 "github.com/openkruise/kruise-api/apps/v1beta1"
+	policyv1alpha1 "github.com/openkruise/kruise-api/policy/v1alpha1"
```

Comment:
Add the kruise prefix before the package name, k8s itself also has a policy package